README.md

```diff
@@ -1,8 +1,18 @@
 Randomizer
 ==========
 
+![screenshot](screenshot.png)
+
 Random symbol sequence (e.g. password) generator for Haiku
 
+Instructions:
+  * Clone the repo and `cd` to the folder where you cloned it and run the following:
```

Comment:
I think ou can be even more specific, as that folder gets cloned and its name therefore known... So just: "Clone the repo and run the following:"
Then in start with "cd Randomizer/src".